js/mixins/download-url.js

```diff
@@ -80,22 +68,23 @@ Highcharts.downloadURL = function (dataURL, filename) {
             throw new Error('Failed to convert to blob');
         }
     }
-
     // Try HTML5 download attr if supported
     if (a.download !== undefined) {
         a.href = dataURL;
         a.download = filename; // HTML5 download attribute
         doc.body.appendChild(a);
         a.click();
         doc.body.removeChild(a);
-    } else {
+    }
+    else {
         // No download attr, just opening data URI
         try {
             windowRef = win.open(dataURL, 'chart');
             if (windowRef === undefined || windowRef === null) {
                 throw new Error('Failed to open window');
             }
-        } catch (e) {
+        }
```

Comment:
Is it possible to configure TypeScript to avoid adding newlines for `else` and `catch` statements?